djangocms_audio/templates/djangocms_audio/default/file.html

```diff
@@ -0,0 +1,16 @@
+{% load i18n %}
+
+<audio controls
```

Comment:
would prefer not to force controls, rather have them enabled by default, but is ok either way